Flaky test_load_balancing_default

https://s3.amazonaws.com/clickhouse-test-reports/49718/56ea17c922c008fc3804770a19688e153daa8c7f/integration_tests__asan__[1_6].html

```
    def test_load_balancing_default():
        unique_nodes = set()
        for _ in range(0, queries):
            unique_nodes.add(get_node(n1, settings={"load_balancing": "random"}))
>       assert len(unique_nodes) == nodes, unique_nodes
E       AssertionError: {'n2', 'n3'}
E       assert 2 == 3
E        +  where 2 = len({'n2', 'n3'})

test_distributed_load_balancing/test.py:129: AssertionError
```

```
=================================== FAILURES ===================================
_________________ test_distributed_replica_max_ignored_errors __________________
[gw4] linux -- Python 3.10.12 /usr/bin/python3

    def test_distributed_replica_max_ignored_errors():
        settings = {
            "use_hedged_requests": 0,
            "load_balancing": "in_order",
            "prefer_localhost_replica": 0,
            "connect_timeout": 2,
            "receive_timeout": 2,
            "send_timeout": 2,
            "idle_connection_timeout": 2,
            "tcp_keep_alive_timeout": 2,
            "distributed_replica_max_ignored_errors": 0,
            "distributed_replica_error_half_life": 60,
        }
    
        # initiate connection (if started only this test)
        n2.query("SELECT * FROM dist", settings=settings)
        cluster.pause_container("n1")
    
        # n1 paused -- skipping, and increment error_count for n1
        # but the query succeeds, no need in query_and_get_error()
        n2.query("SELECT * FROM dist", settings=settings)
        # XXX: due to config reloading we need second time (sigh)
        n2.query("SELECT * FROM dist", settings=settings)
        # check error_count for n1
        assert (
            int(
                n2.query(
                    """
        SELECT errors_count FROM system.clusters
        WHERE cluster = 'replicas_cluster' AND host_name = 'n1'
        """,
                    settings=settings,
                )
            )
            == 1
        )
    
        cluster.unpause_container("n1")
        # still n2
>       assert get_node(n2, settings=settings) == "n2"
E       AssertionError: assert 'n3' == 'n2'
E         - n2
E         + n3
```

--- a/tests/integration/test_distributed_load_balancing/test.py
+++ b/tests/integration/test_distributed_load_balancing/test.py
@@ -200,7 +200,6 @@ def test_distributed_replica_max_ignored_errors():
         "connect_timeout": 2,
         "receive_timeout": 2,
         "send_timeout": 2,
-        "idle_connection_timeout": 2,
         "tcp_keep_alive_timeout": 2,
         "distributed_replica_max_ignored_errors": 0,
         "distributed_replica_error_half_life": 60,

